### PR TITLE
show all avatars

### PR DESCRIPTION
**Pull request by Nyameliaaaa**
adds the ability to see both the user and guild avatar of a member

### CODE REVIEW

**Review by hkubus — DISMISSED** [review]

File: src/components/avatar.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { MessageActionRow, MessageButton, ButtonInteraction, InteractionReplyOptions } from 'discord.js';
+
+export async function run(client: FluorineClient, interaction: ButtonInteraction, value: string) {
+    const [memberID, action] = value.split('.');
```
Comment (SuperchupuDev) [inline src/components/avatar.ts]: can you rename this to `memberId`?

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]

File: src/components/avatar.ts (hunk 1 above)
Comment (Nyameliaaaa) [reply src/components/avatar.ts]: ah camelCase and the word ID, never gets old.

**Review by hkubus — DISMISSED** [review]

File: src/commands/avatar.ts [hunk 2]
```diff
@@ -1,16 +1,66 @@
...
+                    .setCustomId(`avatar:${interaction.user.id}:${member.id}.user`)
+                    .setLabel(client.i18n.t('AVATAR_USER'))
+                    .setStyle('PRIMARY')
+            );
+
+            replyOptions.components = [row];
+        }
+
+        interaction.reply(replyOptions);
+    } else if (user) {
```
Comment (SuperchupuDev) [inline src/commands/avatar.ts]: user will always be true

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]
also remember to add `{ lng: interaction.locale }` to all translations

File: src/commands/avatar.ts (hunk 2 above)
Comment (Nyameliaaaa) [reply src/commands/avatar.ts]: we're checking if we have a user object BUT that user object isnt a guild member
if true then we show only the user avatar (User objects dont have guild avatars)
if user is false we skip into the show the author block.

**Review by SuperchupuDev — DISMISSED** [review]

**Review by SuperchupuDev — APPROVED** [review]

**Review by hkubus — APPROVED** [review]
sent from ur mom

**Pull request merged**